### DISCUSSION TITLE
Retirment of VS Mac

### DISCUSSION THREAD

**Discussion by mrc-eu** [unoplatform/uno#13467] [General]
Sigh.

https://devblogs.microsoft.com/visualstudio/visual-studio-for-mac-retirement-announcement/

**Comment by sasakrsmanovic** [comment]
Hello Marc, 

We've seen the announcement and like many are disappointed. But at the same time we hope its for the better. 

Long time ago we've invested in our VS Code plugin as frankly VS Mac has always been behind VS Windows. While not ideal replacement, we will be evolving it alongside Microsoft's Dev Kit. [https://marketplace.visualstudio.com/items?itemName=unoplatform.vscode](https://marketplace.visualstudio.com/items?itemName=unoplatform.vscode)

It has commonly used XAML Completion (aka Intelisense), debugging, hot reload etc.
